### ISSUE TITLE
Imageboard app crashes when adding a new image to an imageboard

### ISSUE THREAD

**Issue by DanielRosenwasser** [Microsoft/TypeScriptSamples#59]
From https://github.com/Microsoft/TypeScriptSamples/issues/57#issuecomment-157665725

> When I try to add a new Image the app crash with the following error:

> ```
{ result: { ok: 1, n: 1 },
  ops: 
   [ { user: 'lukeh',
       caption: 'COOL',
       imageUri: 'http://ansolas.de/assets/image/logo.svg',
       link: 'http://ansolas.de/assets/image/logo.svg',
       board: 'Flowers That I like',
       comments: [],
       _id: 564c4e1f903c20190471112f } ],
  insertedCount: 1,
  insertedIds: [ 564c4e1f903c20190471112f ] }
[TypeError: Cannot read property '_id' of undefined]
```

**Comment by hermanfransen** [comment]
I have the same problem, did you find a solution?

**Comment by DanielRosenwasser** [comment]
Nope, we've been pretty busy with TS 1.8, but if you're willing to help look into it we'd appreciate it :smiley:

**Comment by hermanfransen** [comment]
Hi Daniel,

I am a newbie, I will try to find the bug.

Thanks.

Kind regards,
Herman Fransen


2016-01-20 21:11 GMT+01:00 Daniel Rosenwasser <notifications@github.com>:
